src/views/GovernanceOverview/index.tsx

```diff
@@ -0,0 +1,165 @@
+import React from 'react';
+import { withRouter, Link } from 'react-router-dom';
+import Header from 'components/Header';
+import './index.scss';
```

Comment:
Relative imports should come after absolute imports. ESLint should have caught this.